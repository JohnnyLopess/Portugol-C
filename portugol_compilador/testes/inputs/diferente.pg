programa{
    inicio
        se (5 != 2){
            escreva("os valores são diferentes")
        } senao {
            escreva("os valores são iguais")
        }
}