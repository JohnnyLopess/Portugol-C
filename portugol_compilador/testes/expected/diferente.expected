#include <stdio.h>
#include <string.h>

int main() {
    if (5 != 2) {
        printf("os valores são diferentes");
    } else {
        printf("os valores são iguais");
    }
    return 0;
}